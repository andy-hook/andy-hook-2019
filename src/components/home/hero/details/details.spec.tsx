import React from "react"
import { renderWithTheme } from "@test-utils"

import Details from "./details"

describe("<Details />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <Details
        buttonHref="https://www.google.com"
        visible={true}
        animate={false}
      />
    )
    expect(tree).toMatchSnapshot()
  })
})
