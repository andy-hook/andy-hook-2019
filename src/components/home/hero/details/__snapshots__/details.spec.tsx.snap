// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Details /> renders correctly 1`] = `
<div
  className="sc-htpNat kQZvsG"
>
  <h2
    className="sc-bxivhb nHtTB"
  >
    <div
      className="sc-bZQynM bOdhzH"
    >
      <div
        className="sc-bZQynM bOdhzH"
      >
        I
      </div>
       
    </div>
    <div
      className="sc-bZQynM bOdhzH"
    >
      <div
        className="sc-bZQynM bOdhzH"
      >
        d
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        e
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        s
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        i
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        g
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        n
      </div>
       
    </div>
    <div
      className="sc-bZQynM bOdhzH"
    >
      <div
        className="sc-bZQynM bOdhzH"
      >
        &
      </div>
       
    </div>
    <div
      className="sc-bZQynM bOdhzH"
    >
      <div
        className="sc-bZQynM bOdhzH"
      >
        b
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        u
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        i
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        l
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        d
      </div>
       
    </div>
    <div
      className="sc-bZQynM bOdhzH"
    >
      <div
        className="sc-bZQynM bOdhzH"
      >
        b
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        e
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        a
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        u
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        t
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        i
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        f
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        u
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        l
      </div>
       
    </div>
    <div
      className="sc-bZQynM bOdhzH"
    >
      <div
        className="sc-bZQynM bOdhzH"
      >
        d
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        i
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        g
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        i
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        t
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        a
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        l
      </div>
       
    </div>
    <div
      className="sc-bZQynM bOdhzH"
    >
      <div
        className="sc-bZQynM bOdhzH"
      >
        i
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        n
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        t
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        e
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        r
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        f
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        a
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        c
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        e
      </div>
      <div
        className="sc-bZQynM bOdhzH"
      >
        s
      </div>
      
    </div>
  </h2>
  <a
    className="sc-bdVaJa hwoiCO"
    href="https://www.google.com"
    onClick={[Function]}
    target="_blank"
  >
    <span
      className="sc-bwzfXH kItmmw"
    >
      View my latest projects
    </span>
  </a>
</div>
`;
